test-packages/test-utils/src/utils/semantic-model-provider.ts

```diff
@@ -253,29 +349,58 @@ const libraryFixes: Record<TestModelVersion, Record<string, LibraryFix[]>> = {
       },
     ],
   },
-  "1.71.14": {},
-  "1.74.0": {
-    "sap.ui.generic.app": [
+  "1.71.49": {},
+  "1.84.27": {},
+  "1.96.11": {
+    "sap.ui.mdc": [
       (content: Json): void => {
-        // Removing from this library. There is another symbol with the same name in library "sap.fe".
-        remove(
-          get(content, "symbols"),
-          (symbol) =>
-            get(symbol, "name") ===
-            "sap.ui.generic.app.navigation.service.NavigationHandler"
-        );
+        forEach(get(content, "symbols"), (symbol) => {
+          const defaultAggregation =
+            symbol?.["ui5-metadata"]?.defaultAggregation;
+          if (
+            defaultAggregation &&
+            !symbol["ui5-metadata"].aggregations?.[defaultAggregation]
+          ) {
+            symbol["ui5-metadata"].aggregations =
+              symbol["ui5-metadata"].aggregations || [];
+            symbol["ui5-metadata"].aggregations.push({
+              name: "content",
+              singularName: "content",
+              type: "sap.ui.core.Control",
+              cardinality: "0..1",
+              visibility: "public",
+              methods: ["getContent", "destroyContent", "setContent"],
+            });
+          }
+        });
       },
     ],
-    "sap.ushell": [
+  },
+  "1.105.0": {
+    "sap.ui.mdc": [
       (content: Json): void => {
```

Comment:
is this copy / pasta? extract to a utility?

Reply:
I prefer to keep the whole version `fixes` and `libraryFixes` atomic as ideally, they should be cleaned-up in the original codebase. I will try to incorporate that checks into the UI5 api.json generation to ensure that these issues will disappear and hopefully no fixes and libraryFixes will be needed anymore.